src/context/ActionSidebarContext/ActionSidebarContextProvider.tsx

```diff
@@ -51,7 +53,12 @@ const ActionSidebarContextProvider: FC<PropsWithChildren> = ({ children }) => {
 
   actionSidebarUseRegisterOnBeforeCloseCallback((element) => {
     const isClickedInside = isElementInsideModalOrPortal(element);
-    if (!isClickedInside) {
+    const navigationWrapper = document.querySelector('.modal-blur-navigation');
```

Comment:
Can we please refactor `document.querySelector` in a utils function?
```
const getElementWithSelector = (selector) => document.querySelector(selector);
```
and use it here `const navigationWrapper = getElementWithSelector('.modal-blur-navigation')`

Reply:
What is the advantage of doing so?

Reply:
The advantage would be to abstract the way elements are selected. 
In case additional logic will be needed in the future, eg. log an error if the element is not found for a specific selector, only the utility function will need to be updated.